env/requirements.yaml

```diff
@@ -33,12 +33,8 @@ dependencies:
   repository: https://charts.gitlab.io/
   version: 0.1.12
 - name: jx-app-ui
-  repository: http://chartmuseum.jenkins-x.io
-  version: 0.1.211
-- name: jx-app-ui
-  repository: http://chartmuseum.jenkins-x.io
-  version: 0.1.211
-  alias: jxuisso
+  repository: https://chartmuseum-jx.jenkins-x.live
```

Comment:
ah, sorry, I copied the wrong chart repo URL.
That is the right one: https://charts.cloudbees.com/cjxd/cloudbees
Could you update it again?